src/steppy/preprocessing/misc.py

```diff
@@ -34,10 +34,9 @@ def save(self, filepath):
 
 
 class ImageReader(BaseTransformer):
-    def __init__(self, x_columns, y_columns, target_shape):
+    def __init__(self, x_columns, y_columns):
```

Comment:
@apyskir I agree think it could be usefull to have the reshape option in the reader especially since there are some libs that facilitate read+resize that are really fast (fastjpeg or something if I remember correctly). That being said target_shape seems to be obsolete in this implementation.

Reply:
@jakubczakon I removed it, because reshape was done twice, actually: once in the reader, and once in `self.image_transform` in loader.

Reply:
@apyskir ok